docs/installation.rst

```diff
@@ -80,6 +80,7 @@ on the releases_ page for the following platforms:
   glibc versions also work, if they are compatible to 2.13.
 * **Mac OS X**: 10.10 (does not work with older OS X releases)
 * **FreeBSD**: 10.2 (unknown whether it works for older releases)
+* **Windows**: Tested on Windows10. (Should work on Vista and up)
```

Comment:
hmm, you did not make a pyinstaller binary, did you? see line 76. So maybe move this to a separate section. later we can try do make a pyinstaller binary.